### ISSUE TITLE
Luma theme CSS results in skewed logo in mobile

### ISSUE THREAD

**Issue by thdoan** [magento/magento2#5293]
## Steps to reproduce
1. Install Magento 2.0.7 (or later).
2. Choose Luma theme.
3. Use a logo where the width and height are the same.
4. View in mobile (or resize window to force mobile view).
## Expected result
1. The logo should be resized proportionately to fit inside the menu bar in mobile:

![screenshot_20160628102124](https://cloud.githubusercontent.com/assets/4110567/16401871/8a96595a-3d1a-11e6-9446-43ecfa45659a.png)
## Actual result
1. The logo is not resized correctly, resulting in a skewed logo:

![screenshot_20160628101019](https://cloud.githubusercontent.com/assets/4110567/16401696/2ab0992a-3d19-11e6-9050-406d26928eae.png)

**Comment by thdoan** [comment]
To fix this, I added `width: auto;` to the following block in styles-m.css:

```
.logo img {
  display: block;
  max-height: 50px;
  max-width: 100%;
  width: auto;
}
```

**Comment by elenleonova** [comment]
Hi, the issue has been fixed in 2.1.0 release - internal issue MAGETWO-46716, thus we're closing the ticket. if you have any other questions, please feel free to reopen or create another ticket.
